ietf-mpls-te-types.yang

```diff
@@ -75,19 +75,16 @@ module ietf-mpls-te-types {
       enum per-flow {
         description
           "The load-balancing algorithm ensures that packets for
-           the same flow are forwarded on the same component link,
-           while packets for different flows can be forwarded on
-           different component links.
+           the same flow are forwarded on the same component link.
 
            Packets for different flows within the same LSP can be
            forwarded on different component links.";
```

Comment:
How about:
The load-balancing algorithm ensures that packets characterized as the same flow (e.g. based on IP 5-tuple) that egress on a LAG or a bundled TE link are forwarded on the same component link.

Reply:
See https://github.com/tsaad-dev/te/pull/104/commits/6565ce7d83432bd76208fa765f56cb991ee6b4fe